autrainer/core/utils/bookkeeping.py

```diff
@@ -12,10 +12,11 @@
 import yaml
 
 from autrainer.metrics import AbstractMetric
+from autrainer.models.utils import create_model_inputs
 
 
 if TYPE_CHECKING:
```

Comment:
claim your free + 0.00001% coverage :)
```suggestion
if TYPE_CHECKING:  # pragma: no cover
```